src/DynamoCoreWpf/UI/GuidedTour/StepUIAutomation.cs

```diff
@@ -63,5 +64,15 @@ public enum UIControlType { MENUITEM, BUTTON, FUNCTION };
         /// </summary>
         public string WindowName { get; set; }
 
+        public List<AutomaticHandlers> AutomaticHandlers { get; set; }
+    }
+
+    public class AutomaticHandlers
```

Comment:
Please add comments for all public members if they indeed need to be/ expected to be public

Reply:
Sorry, comments added.